pkg/landscaper/controllers/deployitem/reconcile_new.go

```diff
@@ -160,11 +161,12 @@ func (con *controller) isAbortingTimeoutExceeded(log logging.Logger, di *lsv1alp
 	return false, &requeue, nil
 }
 
-func (con *controller) writeAbortingTimeoutExceeded(ctx context.Context, log logging.Logger, di *lsv1alpha1.DeployItem) error {
+func (con *controller) writeAbortingTimeoutExceeded(ctx context.Context, di *lsv1alpha1.DeployItem) error {
 	// deploy item has not been aborted within the timeframe
 	// => aborting timeout
-	logger := log.WithValues("operation", "AbortingTimeoutExceeded")
-	logger.Debug("aborting timeout occurred")
+	logger, ctx := logging.FromContextOrNew(ctx, nil)
+	logger = logger.WithValues(lc.KeyMethod, "writeAbortingTimeoutExceeded")
```

Comment:
```suggestion
	logger, ctx := logging.FromContextOrNew(ctx, nil, lc.KeyMethod, "writeAbortingTimeoutExceeded")
```

Reply:
Again the `KeyMethod` logging key, which is only for the current method, but should not be passed in a context to sub methods.